osm-fudge/jaccard.py

```diff
@@ -0,0 +1,20 @@
+'''Jaccard similarity function
+
+Find jaccard distance between two given strings
+
+More: https://en.wikipedia.org/wiki/Jaccard_index
+'''
+
+import sys
+import n_gram
+
+def find_jaccard_similarity(n, str1, str2):
+    a = n_gram.find_n_grams(n, str1)
+    b = n_gram.find_n_grams(n, str2)
+    common = list(set(a) & set(b))
+    union = list(set(a) | set(b))
+    return len(common) / len(union)
```

Comment:
If you want to you could do it more efficiently. At the moment you need to calculate the intersection and the union of two sets to calculate `len(intersection) / len(union)`. But you can re-write this formula into

    len(intersection) / (len(str1) + len(str2) - len(intersection))

Then you don't need to calculate the union at all.